### PR TITLE
Docs fixes 2

### PR DESCRIPTION
**Pull request by beczkowb**

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Deployment
 
 One of the key features of BigFlow is the full automation of the build and deployment process.
-BigFlow can dockerize your workflows and deploy them to Google Cloud Composer.
+BigFlow dockerize your workflows and deploy them to Google Cloud Composer.
```
Comment (bartoszwalacik) [inline docs/deployment.md:4]: dockerizes

File: docs/monitoring.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Monitoring
+
+The easiest way to monitor your processes is to use Cloud Composer [DAG metrics](https://cloud.google.com/composer/docs/monitoring-dashboard?&_ga=2.137435509.-47150562.1597392349#dag_runs).
```
Comment (bartoszwalacik) [inline docs/monitoring.md:3]: bez query stringa w linku

File: docs/monitoring.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+To set up an alert for a metric, go to the Cloud Composer dashboard and open the "Monitoring" section:
```
Comment (bartoszwalacik) [inline docs/monitoring.md:5]: "Monitoring" - without quotes

File: docs/project_structure_and_build.md [hunk 4]
```diff
@@ -198,10 +199,11 @@ from a Docker environment.
 Run the `bigflow build-image` command inside the [`docs`](../docs) project. Next, you can run the example workflow using Docker:
 
 ```shell script
-docker run bigflow-docs:0.1.0 bigflow run --job hello_world_workflow.hello_world
+docker load -i .image/image-{Generated image version}.tar
+docker run {Your loaded image ID} bigflow run --job hello_world_workflow.hello_world --project-package examples
 ```
 
-DAGs generated by BigFlow use KubernetesPodOperator to call this Docker command.
```
Comment (bartoszwalacik) [inline docs/project_structure_and_build.md:204]: this command

File: docs/deployment.md (hunk 1 above)
Comment (beczkowb) [reply docs/deployment.md]: done

File: docs/monitoring.md (hunk 2 above)
Comment (beczkowb) [reply docs/monitoring.md]: done

File: docs/monitoring.md (hunk 3 above)
Comment (beczkowb) [reply docs/monitoring.md]: done

File: docs/project_structure_and_build.md (hunk 4 above)
Comment (beczkowb) [reply docs/project_structure_and_build.md:204]: done

**Pull request merged**